src/ConfigManager.cpp

```diff
@@ -235,13 +268,34 @@ void ConfigManager::setup() {
     startAP();
 }
 
+void ConfigManager::reset() {
+    // char magic[2];
+    char ssid[32];
+    char password[64];
+
+    for (int i = 0; i < sizeof(ssid); i++) {
+        ssid[i] = 0;
+    }
+    
+    for (int i = 0; i < sizeof(password); i++) {
+        password[i] = 0;
+    }
+
+    EEPROM.put(0, 0);
+    EEPROM.put(WIFI_OFFSET, ssid);
+    EEPROM.put(WIFI_OFFSET + 32, password);
+    EEPROM.commit();
+
+    setup();
```

Comment:
This could potentially cause an issue, depending one when it is called. There may already be a WebServer setup on the port and it is never closed. I would suggest not calling this, and documenting the function to just reset the wifi credentials.

Reply:
Id suggest making an endpoint for the reset, versus having a code condition know when to reset it

Reply:
It is a pretty destructive thing to do. Not sure it is great to expose it by default. It would be easy enough to attach it to the API using the callback.

Reply:
True, my original work around was just changing the password in the UI :D